### PR TITLE
Take the  field from HTML and stick it in the import JSON

### PR DESCRIPTION
**Pull request by ryanpbrewster**
@andrewconner look all right?

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/controllers/website/BookmarkImporter.scala [hunk 1]
```diff
@@ -170,13 +170,15 @@ class NetscapeBookmarkParser @Inject() () extends ImportParser with ZipParser {
         val lists = Option(elem.attr("list")).getOrElse("")
         val tags = Option(elem.attr("tags")).getOrElse("")
 
-        val tagList = (lists + tags).split(",").map(_.trim).filter(_.nonEmpty).toList.distinct
+        val tagList = List(lists, tags).flatMap(_.split(',').toList.map(_.trim).filter(_.nonEmpty)).distinct
```
Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/website/BookmarkImporter.scala:173]: I think this had incorrect behavior if, for some reason, both `lists` and `tags` were non-empty.

Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/website/BookmarkImporter.scala:173]: Ah, good catch. Fortunately, doesn't happen (they're each used in different formats)

**Comment by andrewconner** [comment]
Yep this is it. :+1:

**Comment by ryanpbrewster** [comment]
:boom:, just tested and it works. Thanks for the help!

**Pull request merged**